test/images/pets/zookeeper-installer/on-start.sh

```diff
@@ -56,7 +56,7 @@ echo "" > "${CFG_BAK}"
 i=0
 LEADER=$HOSTNAME
 for peer in "${PEERS[@]}"; do
-    let i=i+1
+    (( i=i+1 ))
```

Comment:
Why I change this:

```
In ./test/images/pets/zookeeper-installer/on-start.sh line 59:
    let i=i+1
    ^-------^ SC2219: Instead of 'let expr', prefer (( expr )) .
```